test/e2e/github_release_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGithubRelease(t *testing.T) {
+	env := BuildEnv(t)
+	logger := Logger{}
+	vendir := Vendir{t, env.BinaryPath, logger}
+
+	dstPath, err := os.MkdirTemp("", "vendir-e2e-github-release-dst")
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
```suggestion
	require.NoError(t, err)
```

This change is just to keep the pattern we are using consistent